domain/data.go

```diff
@@ -3,4 +3,5 @@ package domain
 import "fmt"
 
 var UserExistError = fmt.Errorf("user with this login already exists")
```

Comment:
I think, all there errors should be in app, because they are unused in domain, and belongs only to application behavior, not to behaviour of domain entities.
But still not sure...